pallene/syntax_errors.lua

```diff
@@ -55,8 +55,8 @@ local errors = {
     AssignImport = "Expected '='.",
 
     --this label is not thrown in rule 'import' because rule 'toplevelvar'
-    --matches an input like "local bola = X", given that X is a valid expression,
-    --or throws a label when X is not a valid expression
+    --matches an input like "local bola = X", given that X is a valid
```

Comment:
Let's add spaces after the `--` here.

Reply:
Here I just followed the original comment. Do I change all the comments in this file to add spaces after all the `--`?